ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/TransactionLogsIndexer.java

```diff
@@ -117,16 +128,68 @@ private long fillCacheFile(
       if (maybeHeader.isEmpty()) {
         break;
       }
-      final byte[] logs = maybeHeader.get().getLogsBloom().toArray();
-      checkNotNull(logs);
-      checkState(logs.length == 256, "BloomBits are not the correct length");
-      fos.write(logs);
+      fillCacheFileWithBlock(maybeHeader.get(), fos);
       indexingStatus.currentBlock = blockNum;
       blockNum++;
     }
     return blockNum - startBlock;
   }
 
+  public void cacheLogsBloomForBlockHeader(final BlockHeader blockHeader) {
+    try {
+      final long blockNumber = blockHeader.getNumber();
+      LOG.info("Caching logs bloom for block {}.", "0x" + Long.toHexString(blockNumber));
```

Comment:
This should be `debug` - it happens way too often.

Reply:
Done.